### PR TITLE
python@3.8 3.8.1

### PR DESCRIPTION
**Pull request by aiiie**
Created with `brew bump-formula-pr`.

### CODE REVIEW

File: Formula/python@3.8.rb [hunk 1]
```diff
@@ -1,8 +1,8 @@
 class PythonAT38 < Formula
```
Comment (brodie) [inline Formula/python@3.8.rb]: This needs to be changed to PythonAT381. I posted this comment before but it seems to have disappeared.

**Comment by brodie** [comment]
What's the usual workflow for replacing python3 with the latest 3 release? Is it automated?

**Comment by brodie** [comment]
Should the python@3.8 package be renamed to python@3.8.1?

**Comment by SMillerDev** [comment]
No, usually there isn't even a python@3.8. Only the URL and hash should change.

**Comment by MikeMcQuaid** [comment]
Please update https://github.com/Homebrew/homebrew-core/blob/master/Formula/python@3.8.rb instead.

**Comment by brodie** [comment]
I'll update 3.8 then.

Is there a workflow for getting 3.8 into the python3 formula?

**Comment by SMillerDev** [comment]
Yeah, currently people are working on making 3.8 the default python formula. Not everything is compatible yet though.

**Comment by brodie** [comment]
Is there anything I can do to help?

**Comment by iMichka** [comment]
Right now, not much.

The python 3.8 migration is being done here: #47326
We triggered a new build which should start soon.

Helping fix some test failures would help:
https://github.com/Homebrew/homebrew-core/pull/47326#issuecomment-568164979
There are also some other test failures that need to be investigated:
https://github.com/Homebrew/homebrew-core/issues/46728

You can also sponsor us: https://github.com/sponsors/Homebrew
This may help us buy more build machines (or faster ones).

**Comment by fxcoudert** [comment]
Opening a new PR to track this: https://github.com/Homebrew/homebrew-core/pull/48304
Thanks @brodie

**Pull request closed without merging**